scripts/mas-sso-config.sh

```diff
@@ -0,0 +1,113 @@
+kasClientId=$(jq -r '.[].id' <<< $KAS)
+
+#/auth/admin/realms/rhoas/clients/de121cf7-a6b2-4d39-a99c-8da787454a66/service-account-user
+SVC=`curl -sk --header "Content-Type: application/json" --header "Authorization: Bearer $TOKEN" $KEYCLOAK_URL/auth/admin/realms/rhoas/clients/$kasClientId/service-account-user`
+svcUserId=$(jq -r '.id' <<< $SVC)
+echo $svcUserId
+
+FINAL=`curl -sk --data-raw '[{"id": '$manageUser',"name": "manage-users"},{"id": '$manageRealm',"name": "manage-realm"},{"id": '$manageClients',"name": "manage-clients"}]' --header "Content-Type: application/json" --header "Authorization: Bearer $TOKEN" $KEYCLOAK_URL/auth/admin/realms/rhoas/users/$svcUserId/role-mappings/clients/$realmMgmtClientId`
+echo $FINAL
+
+
+
+CREATE=`curl -sk --data-raw '{
+   "authorizationServicesEnabled": false,
+   "clientId": "kas-fleet-manager",
+   "description": "kas-fleet-manager",
+   "name": "kas-fleet-manager",
+   "secret":"kas-fleet-manager",
+    "directAccessGrantsEnabled": false,
+    "serviceAccountsEnabled": true,
+    "publicClient": false,
+    "protocol": "openid-connect"
+}' --header "Content-Type: application/json" --header "Authorization: Bearer $TOKEN" $KEYCLOAK_URL/auth/admin/realms/rhoas-kafka-sre/clients`
+echo $CREATE
+
+RE=`curl -sk --header "Content-Type: application/json" --header "Authorization: Bearer $TOKEN" $KEYCLOAK_URL/auth/admin/realms/rhoas-kafka-sre/clients?clientId=realm-management`
+realmMgmtClientId=$(jq -r '.[].id' <<< $RE)
+echo $realmMgmtClientId
+
+
+ROLES=`curl -sk --header "Content-Type: application/json" --header "Authorization: Bearer $TOKEN" $KEYCLOAK_URL/auth/admin/realms/rhoas-kafka-sre/clients/$realmMgmtClientId/roles`
+manageUser=$(jq -c '.[] | select( .name | contains("manage-users")).id' <<< $ROLES)
+manageClients=$(jq -c '.[] | select( .name | contains("manage-clients")).id' <<< $ROLES)
+manageRealm=$(jq -c '.[] | select( .name | contains("manage-realm")).id' <<< $ROLES)
+echo $manageUser
+echo $manageRealm
+echo $manageClients
+
+
+KAS=`curl -sk --header "Content-Type: application/json" --header "Authorization: Bearer $TOKEN" $KEYCLOAK_URL/auth/admin/realms/rhoas-kafka-sre/clients?clientId=kas-fleet-manager`
+kasClientId=$(jq -r '.[].id' <<< $KAS)
+
+#/auth/admin/realms/rhoas/clients/de121cf7-a6b2-4d39-a99c-8da787454a66/service-account-user
+SVC=`curl -sk --header "Content-Type: application/json" --header "Authorization: Bearer $TOKEN" $KEYCLOAK_URL/auth/admin/realms/rhoas-kafka-sre/clients/$kasClientId/service-account-user`
+svcUserId=$(jq -r '.id' <<< $SVC)
+echo $svcUserId
+
+FINAL=`curl -sk --data-raw '[{"id": '$manageUser',"name": "manage-users"},{"id": '$manageRealm',"name": "manage-realm"},{"id": '$manageClients',"name": "manage-clients"}]' --header "Content-Type: application/json" --header "Authorization: Bearer $TOKEN" $KEYCLOAK_URL/auth/admin/realms/rhoas-kafka-sre/users/$svcUserId/role-mappings/clients/$realmMgmtClientId`
```

Comment:
$( .. ) syntax should be preferred to back tick syntax (https://pubs.opengroup.org/onlinepubs/9699919799/xrat/V4_xcu_chap02.html#tag_23_02_06_03)

Reply:
Thanks for the suggestion. we can improve this later. Anyways this part will be changed with migration.